src/lang/en-US.ts

```diff
@@ -32,7 +32,7 @@ const CommonMessages: ResolvedIntlConfig['messages'] = {
     'common.noUnDo': `This action cannot be undone.`,
     'common.version': `version`,
     'common.tenant': `Prefix`,
-    'common.tenant.creationForm': `Name`,
+    'common.tenant.creationForm': `Organization`,
```

Comment:
A lot of people have submitted their own name here so trying to make it clear we want an organization's name and not their own name.